src/components/endpoints/SendTransaction.vue

```diff
@@ -30,7 +30,8 @@ const isPending = ref(false)
 
 const sampleData = computed<{ [key: string]: TransactionSelect[] }>(() => {
   const sampleEoA = '0x311611C9A46a192C14Ea993159a0498EDE5578aC'
-  const sampleUP = '0x4658F1Ac64486827f59E637bE9800Eb035b6f43C'
+  const sampleUP = '0x410f64cdc9c95b19659ccbd35cc5933c20489f6a'
```

Comment:
Let's make sure the address we put here always has a username and a profile picture.

Any reasons for changing the address here?

Reply:
Just tried different one while testing since I've "lost" access to previous one.